src/containers/Bookings.js

```diff
@@ -2,28 +2,89 @@ import React, { Component } from 'react';
 import Search from '../components/Search.js';
 import Results from '../components/Results.js';
 import fakeBookings from '../data/fakeBookings.json';
+import Counter from '../components/Counter';
+//import addClickHandler from '../component/addClickHandler';
 
 export default class Bookings extends Component {
+  state = {
+    results: fakeBookings,
+    Counter:Results,
+    customerIdInput: "",
+    customerNameInput:"",
+   
+   
+  }
+
+  customerInputChange = (event) => {
+    this.setState({
+      customerIdInput: event.target.value
```

Comment:
We discussed in class that this uses the *controlled component* pattern, and how that's slightly unusual for this particular use. We looked at the *uncontrolled component* pattern too - where you use refs to capture the value of the input when clicking the button.